src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedAccessorValueParameterSymbol.cs

```diff
@@ -15,15 +16,24 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// </summary>
     internal sealed class SynthesizedAccessorValueParameterSymbol : SourceComplexParameterSymbol
     {
-        public SynthesizedAccessorValueParameterSymbol(SourceMemberMethodSymbol accessor, TypeWithAnnotations paramType, int ordinal)
+        // Disallow/AllowNull attributes will be added to the `value` parameter if they were present on the property.
+        private readonly FlowAnalysisAnnotations _annotations;
```

Comment:
>private readonly FlowAnalysisAnnotations _annotations; [](start = 8, length = 54)

This field looks unnecessary, the information can easily be calculated/pulled from the property on demand